deploy/backplane/mtsre/10-mtsre-admins-project.ClusterRole.yml

```diff
@@ -0,0 +1,26 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: backplane-mtsre-admins-project
+rules:
+# MT-SRE can portforward pods
+- apiGroups:
+  - ""
+  resources:
+  - pods/portforward
```

Comment:
we need this access to be able to portforward pod/svc

Reply:
Last I checked, port-forwarding also requires get/list "pods" as well.
Idk if you already checked that but thought that I'll mention that here :)

Ref: https://medium.com/@ManagedKube/kubernetes-rbac-port-forward-4c7eb3951e28

Reply:
@yashvardhan-kukreja you are right!  updated

Reply:
Shouldn't this be scope to namespaces the add-ons are using?

Reply:
@fahlmant addons we SRE they are here https://github.com/openshift/managed-cluster-config/tree/master/deploy/backplane/mtsre/managed-odh. this admin-project level access is only for openshift-addon-operator ns. hope it make sense